Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/TicketService.hs

```diff
@@ -145,10 +191,14 @@ getTicketPlacesServices _ placeId mbDate = do
             id = peopleCategory.id,
             pricePerUnit = peopleCategory.pricePerUnit.amount,
             pricePerUnitWithCurrency = mkPriceAPIEntity peopleCategory.pricePerUnit,
-            description = peopleCategory.description
+            description = peopleCategory.description,
+            cancellationCharges = peopleCategory.cancellationCharges
           }
 
-    findSpecialOccasion service = QSO.findAllSpecialOccasionByEntityId service.id.getId Nothing
+    findSpecialOccasion service bookingDate = do
+      allSplOccassions <- QSO.findAllSpecialOccasionByEntityId service.id.getId Nothing
+      allSplOccassionsAtDate <- QSO.findAllSpecialOccasionByEntityId service.id.getId (Just bookingDate)
```

Comment:
Can we have just one call to db here as both tables are same? We can modify the query accordingly

Reply:
it is very small table, and it is not generatable by dsl

Reply:
Table will be become large since for each cancellation, an entry is being created. Check for a way - maybe have array of dates in where clause and always pass  [Nothing, bookingDate]